portfolio/src/main/webapp/script.js

```diff
@@ -30,6 +30,13 @@ function addRandomGreeting() {
 async function fetchRandomNumber() {
   const responseFromServer = await fetch('/random-number');
   const textFromResponse = await responseFromServer.text();
-  const randomContainer = document.getElementById('random-number-container');
-  randomContainer.innerText = textFromResponse;
+  const numberContainer = document.getElementById('random-number-container');
```

Comment:
Way to rename things to be more specific 👍

Reply:
thx c: